doc/README.hive

```diff
@@ -0,0 +1,162 @@
+Instructions For Running Hive
+------------------------------
+
+0) If necessary, download your favorite version of Hive off of Apache
+   and install it into a location where it's accessible on all cluster
+   nodes.  Usually this is on a NFS home directory.  You'll also need to
+   download a binary version of PostgreSQL
+
+   See 'Convenience Scripts' in README about
```

Comment:
perhaps here you should mention hive-testbench.  You can say it's an optional download, but necessary for running several of the benchmarking tests for Hive that come with magpie.

And perhaps say to see below about patching hive-testbench.

Reply:
looks good, thanks!